app/assets/javascripts/student_profile/page_container.js

```diff
@@ -114,8 +114,8 @@
       });
     },
 
-    onSaveServiceFail: function(request, status, message) {
-      this.setState({ requests: merge(this.state.requests, { saveService: 'error' }) });
+    onSaveServiceFail: function(response, status, message) {
+      this.setState({ requests: merge(this.state.requests, { saveService: response.responseJSON.errors[0]}) });
```

Comment:
Cool, that sounds like a good approach too!  Any thoughts you have here that we can capture in the code are great to help future us and future others 👍